docs/book/features/block_volume.md

```diff
@@ -0,0 +1,278 @@
+  key is populated, as is the case below
+
+  The `Status` should say `Bound`. You can also see the `Claim` is set to the above `PersistentVolumeClaim` name `example-vanilla-block-pvc`
+
+```text
+  $ kubectl describe pv pvc-7ed39d8e-7896-11ea-a119-005056983fec
+    Name:            pvc-7ed39d8e-7896-11ea-a119-005056983fec
+    Labels:          <none>
+    Annotations:     pv.kubernetes.io/provisioned-by: csi.vsphere.vmware.com
+    Finalizers:      [kubernetes.io/pv-protection]
+    StorageClass:    example-vanilla-block-sc
+    Status:          Bound
+    Claim:           default/example-vanilla-block-pvc
+    Reclaim Policy:  Delete
+    Access Modes:    RWO
+    VolumeMode:      Filesystem
+    Capacity:        5Gi
+    Node Affinity:   <none>
+    Message:
+    Source:
+        Type:              CSI (a Container Storage Interface (CSI) volume source)
+        Driver:            csi.vsphere.vmware.com
+        VolumeHandle:      e4073a6d-642e-4dff-8f4a-b4e3a47c4bbd
+        ReadOnly:          false
+        VolumeAttributes:      storage.kubernetes.io/csiProvisionerIdentity=1586239648866-8081-csi.vsphere.vmware.com
+                               type=vSphere CNS Block Volume
+    Events:                <none>
+```
+
+### Steps to dynamically provision a block volume in Project Pacific
+
+In the case of `Project Pacific` clusters, `StorageClass` creation happens automatically when a Storage Policy gets created.
+
+The user has to do the following to provision a `PersistentVolume`:
+
+* Create a new storage policy or Add an existing storage policy of interest to the current namespace
+
+* Specify the `StoragePolicy` name (which got created by the storage policy) in the `PersistentVolumeClaim` spec
+
+* Create a `PersistentVolumeClaim` in the namespace using the spec mentioned in the previous step
+
+* Wait for CNS to create a `PersistentVolume`
+
+* Verify if the status of `PersistentVolumeClaim` set to `Bound`
+
+## Static Volume Provisioning
+
+If you have an existing persistent storage device in your VC, you can use static provisioning to make the storage
+instance available to your cluster.
+
+### How does it work
+
+Static provisioning is a feature that is native to Kubernetes and that allows cluster administrators to make existing
+storage devices available to a cluster.
+
+As a cluster administrator, you must know the details of the storage device, its supported configurations, and mount options.
+
+To make existing storage available to a cluster user, you must manually create the storage device, a `PeristentVolume`,
+and a `PersistentVolumeClaim`.
+Because the PV and the storage device already exists, there is no need to specify a storage class name in the PVC spec.
+One of the ways to create the `PV` and the `PVC` binding is by using Label matching.
+
+Static Volume Provisioning is supported only in Vanilla Kubernetes clusters and across guess clusters.
```

Comment:
s/guess/Tanzu

Reply:
Are guest clusters called as `Tanzu Kubernetes Grid` outside VMware?

Reply:
Tanzu Kubernetes Grid is the new name.  See the end of README for 3 flavors:
https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/docs/book/README.md